README.md

```diff
@@ -1,6 +1,16 @@
 # ChromeIDE
 Parallax microcontroller development environment based on Chrome application technologies.
 
+## Quickstart
+
+1. Visit our [github releases](https://github.com/parallaxinc/ChromeIDE/releases) and download the latest release. Or download [v0.1.0 here](https://github.com/parallaxinc/ChromeIDE/releases/download/v0.1.0/chromeide.zip).
```

Comment:
use the link to latest https://help.github.com/articles/linking-to-releases/#linking-to-the-latest-release